Source/Sequel-Ace-Bridging-Header.h

```diff
@@ -35,3 +35,4 @@
 #import <fmdb/FMDB.h>
 #import "SPFileManagerAdditions.h"
 
+@import Firebase;
```

Comment:
Why this? You can simply import Firebase in the swift file?

Reply:
ah ok